v2.0/app/redux/redux-actions/adminActions.js

```diff
@@ -72,7 +72,8 @@ export const reduxGetAllCommunityVendors = (community_id) => dispatch => {
 };
 
 export const reduxGetAllCommunityTestimonials = (community_id) => dispatch => {
-  apiCall('/testimonials.listForCommunityAdmin', { community_id }).then(response => {
```

Comment:
I’ll vote for consistency over inconsistency.  Let’s not have to put the slash at the front